src/applications/disability-benefits/526EZ/components/ConfirmationPoll.jsx

```diff
@@ -0,0 +1,192 @@
+    <p><a href="/track-claims">Check the status of your claim.</a></p>
+    <p>If you don’t see your increased disability claim online after 24 hours, please call Vets.gov Help Desk at <a href="tel:+18555747286">1-855-574-7286</a>, Monday – Friday, 8:00 a.m. – 9:00 a.m. (ET).</p>
+  </div>
+);
+
+const errorMessage = (jobId) => (
+  <div>
+    <p>We’re sorry. Something went wrong on our end when we tried to submit your application. For help, please call the Vets.gov Help Desk at <a href="tel:+18555747286">1-855-574-7286</a>, Monday – Friday, 8:00 a.m. – 9:00 a.m. (ET).</p>
+    <strong>Confirmation number</strong>
+    <div>{jobId}</div>
+    <br/>
+  </div>
+);
+
+// TODO: Make this a loading spinner
+const pendingMessage = (longWait) => {
+  const message = !longWait
+    ? 'Please wait while we submit your application and give you a confirmation number.'
+    : 'We’re sorry. It’s taking us longer than expected to submit your application. Thank you for your patience.';
+  return <LoadingIndicator message={message}/>;
+};
+
+export class ConfirmationPoll extends React.Component {
+  // Using it as a prop for easy testing
+  static defaultProps = {
+    pollRate: 5000
+  }
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      submissionStatus: submissionStatuses.pending,
+      claimId: null,
+      failureCode: null,
+      longWait: false
+    };
+  }
+
+  componentDidMount() {
+    this.__isMounted = true;
+    this.__startTime = Date.now();
+    this.poll();
+  }
+
+  componentWillUnmount() {
+    this.__isMounted = false;
+  }
+
+  poll = () => {
+    // Don't continue to request after the component is unmounted
+    if (!this.__isMounted) {
+      return;
+    }
+
+    apiRequest(`/disability_compensation_form/submission_status/${this.props.jobId}`)
+      .then((response) => {
+        // Don't process the request once it comes back if the component is no longer mounted
+        if (!this.__isMounted) {
+          return;
+        }
+
+        // Check status
+        const status = response.data.attributes.transactionStatus;
+        if (terminalStatuses.includes(status)) {
+          this.setState({
+            submissionStatus: status,
+            claimId: get('data.attributes.metadata.claimId', response) || null
+          });
+        } else {
+          // Wait for a bit and recurse
+          const waitTime = status === submissionStatuses.pending
+            ? this.props.pollRate
+            : this.props.pollRate * 2; // Seems like we don't need to poll as frequently when we get here
+
+          // Force a re-render to update the pending message if necessary
+          if (Date.now() - this.__startTime >= 30000) {
+            this.setState({ longWait: true });
+          }
+          setTimeout(this.poll, waitTime);
+        }
+      })
+      .catch((response) => {
+        // The call to the API failed
+        Raven.captureMessage('526_submission_status_poll_failure', {
```

Comment:
I'm not sure if you need to log this to Sentry. If you get here, it's because the api responded with an error status, which will be logged in vets-api Sentry.

Reply:
I asked Silvio if it was, and he said no. I think he made a PR to change that, now that you mention it.

You're saying all non-okay vets-api responses are logged to Sentry by the api?